csrc/ir/nodes.cpp

```diff
@@ -593,6 +593,7 @@ std::vector<PolymorphicValue> BinaryOp::evaluate(
       return {lhs * rhs};
       break;
     case BinaryOpType::Div:
+    case BinaryOpType::Truediv:
```

Comment:
I don't think this will work. Isn't `2 truediv 5 = 0.4`, instead of `0`?

Reply:
Can we do this?
```cpp
    case BinaryOpType::Truediv:
      return {
          SimplifyingIrBuilder::maybeCastExpr(DataType::Float, lhs) /
          SimplifyingIrBuilder::maybeCastExpr(DataType::Float, rhs)};
      break;
````

Reply:
IIRC, this is what we do in the `truediv` composite op.

Reply:
I just removed the changes to `evaluate`.